### ISSUE TITLE
Error using Junet

### ISSUE THREAD

**Issue by ennvvy** [inguar/Junet.jl#2]
Hi,
   I am new to Junet and installed the package only recently. While doing so, Julia warns about several deprecated commands in it. While using functions like erdosrenyi, it throws an error saying it is not defined. Please suggest what could be done.

Thanks!

**Comment by PedroSapichu** [comment]
Hi @inguar, thanks for Junet.jl, it seems from your talk at JuliaCon that it might be very useful for network research. I was wondering at what stage are you to make it compatible with Julia 0.6.0., because I'm getting lots of errors like @ennvvy, and I haven't been able to test any of the commands you have in the documentation. 

Thanks!,
Pedro

**Comment by inguar** [comment]
Hi @PedroSapichu and @ennvvy! Thank you for bearing with me. I have been busy on other projects lately, but I'm going to revamp the code and fix the errors over this weekend.

**Comment by inguar** [comment]
It has been a while, but this issue is now resolved. I had to rewrite several major parts of the package and now it is fully compatible with Julia 0.6+. Some method names have changed, e.g., the old `erdosrenyi` is now `graph_gilbert` and the actual `graph_erdos_renyi` method is introduced. That properly attributes the models to [their actual inventors](https://en.wikipedia.org/wiki/Erdős–Rényi_model). In case your code uses the old spelling, Julia will show a deprecation warning. 

Additionally, `graph_random` is added, which is an intuitive interface for the previous two models. It selects the correct implementation based on its second argument. If it is a number of edges, `graph_erdos_renyi` is used, and if it's a probability of edges — `graph_gilbert`. I suggest switching to `graph_random` in most cases.

Finally, I'm working on the first ever release, complete with tests and documentation, which will hopefully ship in about two weeks. See you in the coming year! 🎄🎉
